packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -1168,17 +1173,89 @@ class WKWebView extends UIView {
       inspectable,
     );
   }
+}
 
-  /// The custom user agent string.
-  ///
-  /// Represents [WKWebView.customUserAgent](https://developer.apple.com/documentation/webkit/wkwebview/1414950-customuseragent?language=objc).
-  Future<String?> getCustomUserAgent() {
-    return _webViewApi.getCustomUserAgentForInstances(this);
+/// The iOS version of a WKWebView.
+class WKWebViewIOS extends WKWebView implements UIView {
```

Comment:
@bparrishMines This is the other half of how I'm handling the different parenting: the non-abstract classes are these subclasses which are different for the two platforms.